Jenkinsfile

```diff
@@ -77,7 +77,7 @@ pipeline {
 def getStageFromBranch() {
 
     def branchToStageMap = [
-            "origin/intg": "intg",
+            "origin/master": "intg",
```

Comment:
Will it eventually be switched back to intg once we've set up the new branching scheme?

Reply:
I guess so, if that is what we will be doing with the branching